podcasts.xcodeproj/project.pbxproj

```diff
@@ -5873,19 +5883,21 @@
 		BDF9DDDC1B8AE9B900E77B35 /* Extensions */ = {
 			isa = PBXGroup;
 			children = (
-				BDD5253920477E4400AAD211 /* NSObject+AppDelegate.swift */,
-				BDA9668E2189805F00DF9370 /* UITableView+MiniPlayer.swift */,
-				40A0A91024B5FE5300C29362 /* UITableView+MultiSelect.swift */,
-				BD6F1644219AB82700F34D14 /* UISearchBar+TextColor.swift */,
-				BD74F1B521F0133B002EF774 /* MPFeedbackCommand+Title.swift */,
-				407502B5240DCBF200437528 /* NetworkUtils+Helpers.swift */,
-				BDB4E7D1240E3134005770E3 /* String+Differentiable.swift */,
 				BD2CB3E22418B48600E2D762 /* ArrayExtension.swift */,
 				463538AA26E814B300BA9D35 /* DataModel+Strings.swift */,
-				463538AD26E8F4AE00BA9D35 /* Server+Strings.swift */,
+				C7F4BABE28DB7F7C001C9785 /* DiscoverItem+Helper.swift */,
```

Comment:
I think these changes aren't intentional?

Given this PR will go to a feature branch, I have a concern about these changes because it's high likely that `trunk` and the feature branch will diverge and we might run into some nasty merge conflicts.

Or you can cherry-pick these changes and target `trunk`, given they are mainly design changes and not login-flow changes I think that would be ok.